sci-misc/boinc/boinc-7.6.25.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+#WANT_AUTOMAKE="1.11"
+
+AUTOTOOLS_AUTORECONF=true
+
+inherit autotools-utils eutils flag-o-matic systemd user versionator wxwidgets
+
+MY_PV=$(get_version_component_range 1-2)
+
+DESCRIPTION="The Berkeley Open Infrastructure for Network Computing"
+HOMEPAGE="http://boinc.ssl.berkeley.edu/"
+SRC_URI="https://github.com/BOINC/boinc/archive/client_release/${MY_PV}/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="X cuda static-libs"
+
+RDEPEND="
+	!sci-misc/boinc-bin
+	!app-admin/quickswitch
+	>=app-misc/ca-certificates-20080809
+	dev-libs/openssl:0=
+	net-misc/curl[ssl,-gnutls(-),-nss(-),curl_ssl_openssl(+)]
+	sys-apps/util-linux
+	sys-libs/zlib
+	cuda? (
+		>=dev-util/nvidia-cuda-toolkit-2.1
+		>=x11-drivers/nvidia-drivers-180.22
+	)
+	X? (
+		dev-db/sqlite:3
+		media-libs/freeglut
+		sys-libs/glibc:2.2
+		virtual/jpeg:0=
+		x11-libs/gtk+:2
+		>=x11-libs/libnotify-0.7
+		x11-libs/wxGTK:3.0[X,opengl,webkit]
+	)
+"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	app-text/docbook-xml-dtd:4.4
+	app-text/docbook2X
+"
+
+S="${WORKDIR}/${PN}-client_release-${MY_PV}-${PV}"
+
+AUTOTOOLS_IN_SOURCE_BUILD=1
+
+src_prepare() {
+	# prevent bad changes in compile flags, bug 286701
+	sed -i -e "s:BOINC_SET_COMPILE_FLAGS::" configure.ac || die "sed failed"
+
+	autotools-utils_src_prepare
+}
+
+src_configure() {
+	local myeconfargs=(
```

Comment:
get rid of the array, attach everything to `econf` (don't forget to escape the newlines!)